tests/runtest.py

```diff
@@ -705,7 +705,7 @@ def __del__(self):
             sp.call(['rm', '-rf', self.test_dir])
 
 class PyTestBase(TestBase):
-    def __init__(self, name, result, lang='Python', cflags='', ldflags='', sort='simple', serial=False):
+    def __init__(self, name, result, lang='Python', cflags='', ldflags='', sort='task', serial=False):
```

Comment:
I think calling os._exit() is an exceptional case.  Why not just change the upcoming exit test and leave others as is?